assets/js/modules/adsense/index.js

```diff
@@ -88,6 +85,14 @@ addFilter(
 );
 
 domReady( () => {
+	Data.dispatch( CORE_MODULES ).registerModule(
+		'adsense',
+		{
+			settingsEditComponent: SettingsEdit,
+			settingsViewComponent: SettingsView,
+		}
+	);
```

Comment:
Note that #2074 introduces `Modules.registerModule`, which should be used here. To make this smoother, can you add that as part of here too (just copy https://github.com/google/site-kit-wp/pull/2129/files#diff-813cf127a74b2b233ac59519257a5d7c4079deb9707cca867e4c7a5e06d16f40R39) and use it throughout?

Reply:
Sounds good, updated 👍